app/models/site.rb

```diff
@@ -0,0 +1,7 @@
+class Site < ActiveRecord::Base
+  validates :application_name, presence: true, allow_nil: true
```

Comment:
My intent was to require users to set an application title, but allow it to be unset initially.